valhalla/jawn/src/controllers/public/heliconeSqlController.ts

```diff
@@ -63,4 +84,48 @@ export class HeliconeSqlController extends Controller {
     this.setStatus(200);
     return ok(result.data);
   }
+
+  @Get("saved-queries")
+  public async getSavedQueries(
+    @Request() request: JawnAuthenticatedRequest
+  ): Promise<Result<Array<HqlSavedQuery>, string>> {
+    const heliconeSqlManager = new HqlQueryManager(request.authParams);
+    const res = await heliconeSqlManager.getSavedQueries();
+    if (res.error) {
+      this.setStatus(500);
+      return err(res.error);
+    }
+    this.setStatus(200);
+    return ok(res.data || []);
+  }
+
+  @Post("create-saved-query")
```

Comment:
```suggestion
  @Post("saved-query")
```

nitpicky - more "RESTful" but we're not that restful in most places so eh

Reply:
i actually agree here